Sources/Common/DataModel/BoundingBox/index.js

```diff
@@ -129,6 +133,37 @@ export function inflate(bounds, delta) {
   return bounds;
 }
 
+function minInflate(bounds) {
+  const minPnt = [bounds[0], bounds[2], bounds[4]];
+  const maxPnt = [bounds[1], bounds[3], bounds[5]];
+  const nonZero = [0, 0, 0];
+  let maxIdx = -1;
+  let max = 0.0;
+  let w = 0.0;
+  for (let i = 0; i < 3; ++i) {
+    w = maxPnt[i] - minPnt[i];
```

Comment:
`w = bounds[3*i+1]-bounds[3*i]` to avoid building intermediate minPnt and maxPnt arrays.